libs/api/domains/mortgage-certificate/src/lib/graphql/main.resolver.ts

```diff
@@ -0,0 +1,28 @@
+import { Args, Query, Resolver } from '@nestjs/graphql'
+import { UseGuards } from '@nestjs/common'
+import type { User } from '@island.is/auth-nest-tools'
+import {
+  IdsUserGuard,
+  ScopesGuard,
+  CurrentUser,
+} from '@island.is/auth-nest-tools'
+import { MortgageCertificateService } from '../mortgageCertificate.service'
+import { MortgageCertificateValidationModel } from './models'
+import { ValidateMortgageCertificateInput } from './dto/validateMortgageCertificate.input'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Resolver()
+export class MainResolver {
```

Comment:
I added 5 more spaces in front of "@island-is/..", since "/libs/application/template-api-modules/src/lib/modules/templates/mortgage-certificate-submission" is a long line, and I wanted "@island-is/.." to remain in a straight line. Do you want me to change it back?

Then it might look something like this:

![image](https://user-images.githubusercontent.com/6495412/157212508-19cb3c92-0832-4ec8-8d03-a1a457ceff07.png)


Instead of 

![image](https://user-images.githubusercontent.com/6495412/157212581-8cca03ac-7765-43c6-a64d-3c10b1b20b17.png)